### PR TITLE
:bug: Update manage imports table

### PR DESCRIPTION
**Pull request by ibolton336**
Resolves https://issues.redhat.com/browse/MTA-1693
Resolves https://issues.redhat.com/browse/MTA-1716 
- Invalidates queries for import summaries when deleting to trigger a refetch
- Refactors manage imports table to fix mta-1693 issue and use new table format
- Updates template to reflect new fields available for owner / contributor

### CODE REVIEW

**Comment by ibolton336** [comment]
https://github.com/konveyor/tackle2-hub/issues/538

File: client/src/app/pages/applications/manage-imports/manage-imports.tsx [hunk 1]
```diff
@@ -363,12 +206,163 @@ export const ManageImports: React.FC = () => {
...
+                                  }
+                                  footerContent={
+                                    <div>
+                                      <a
+                                        href="https://tackle-docs.konveyor.io/documentation/doc-installing-and-using-tackle/master/index.html#importing-applications_tackle"
+                                        target="_blank"
+                                        rel="noreferrer"
+                                      >
+                                        {t("actions.checkDocumentation")}
+                                      </a>
```
Comment (sjd78) [inline client/src/app/pages/applications/manage-imports/manage-imports.tsx:308]: Could use component `client/src/app/components/ExternalLink.tsx` here.

**Review by sjd78 — COMMENTED** [review]
Looks good to me with one suggestion on the error popover with the external href

**Review by aufi — APPROVED** [review]
CSV content-wise looks good to me.

**Pull request merged**